test/distributed/_tensor/test_dtensor_compile.py

```diff
@@ -276,5 +293,7 @@ def fn(x, y):
         self.assertEqual(y_ref.grad, y.grad)
 
 
+instantiate_parametrized_tests(TestDTensorCompileE2E)
+
 if __name__ == "__main__":
     run_tests()
```

Comment:
Do you want to separate out the PairwiseParallel and SequenceParallel style test cases later?

Reply:
For now they share a lot of in common and I don't see any reason why we are going to split in future.